### ISSUE TITLE
bug(<scope>): <title starting with lowercase letter>

### ISSUE THREAD

**Issue by marcinkrasowski** [Checkmarx/kics#6666]
### Problem description

When there are properties named `content` or `headers`, e.g.
```
User:
  type: object
  properties:
    id:
      type: integer
      format: int64
      minimum: 0
      maximum: 100
    content:                  <--- this
      type: string
      minLength: 0
      maxLength: 20
      pattern: ^([\S\s])*$
    headers:                  <--- this
      type: string
      minLength: 0
      maxLength: 20
      pattern: ^([\S\s])*$
  additionalProperties: false
```
KICS seems to recognize them incorrectly as body and headers of request/response, which results with several false warnings in the `Header Object Without Schema` and `Media Type Object Without Schema` queries. When the fields are renamed into anything else, the warnings disappear.

### Expected Behavior

KICS doesn't return warnings for queries `Header Object Without Schema` and `Media Type Object Without Schema` for properties named `content` or `headers`.

### Actual Behavior

Queries `Header Object Without Schema` and `Media Type Object Without Schema` do not pass successfully for properties named `content` or `headers`.

### Steps to Reproduce the Problem

1. Save [api-incorrect.txt](https://github.com/Checkmarx/kics/files/12416279/api-incorrect.txt) as YAML file (this file contains `content` and `headers` properties in the `User` schema)
3. Run scan on that file, e.g. with `docker run -t -v /mnt/c/workspace:/path checkmarx/kics:latest scan -p /path/api-incorrect.yaml -o "/path/" --exclude-severities 'info,trace'`
4. There will be 15 MEDIUM warnings, which you can see in [results.txt](https://github.com/Checkmarx/kics/files/12416327/results.txt)
5. Save [api-correct.txt](https://github.com/Checkmarx/kics/files/12416278/api-correct.txt) as YAML file  (this file contains `fistName` and `lastName` properties in the `User` schema)
6. Run the same scan (e.g. `docker run -t -v /mnt/c/workspace:/path checkmarx/kics:latest scan -p /path/api-correct.yaml -o "/path/" --exclude-severities 'info,trace'`) on that file and verify that the warnings are gone

### Specifications

-   Version: v1.7.5
-   Platform: Ubuntu
-   Subsystem: Docker

**Comment by kicsbot** [comment]
Please, follow the guideline for an issue title: 

For **bug**:

`bug(<scope>): <title starting with lowercase letter>`

For **query**:

`query(<platform>): <title starting with lowercase letter>`

For **feature request**:

`feat(<scope>): <title starting with lowercase letter>`

Thank you!
*KICS Team*

**Comment by kicsbot** [comment]
Please, follow the guideline for an issue title: 

For **bug**:

`bug(<scope>): <title starting with lowercase letter>`

For **query**:

`query(<platform>): <title starting with lowercase letter>`

For **feature request**:

`feat(<scope>): <title starting with lowercase letter>`

Thank you!
*KICS Team*

**Comment by kicsbot** [comment]
Please, follow the guideline for an issue title: 

For **bug**:

`bug(<scope>): <title starting with lowercase letter>`

For **query**:

`query(<platform>): <title starting with lowercase letter>`

For **feature request**:

`feat(<scope>): <title starting with lowercase letter>`

Thank you!
*KICS Team*
